target/linux/lantiq/files/arch/mips/boot/dts/lantiq/vr9_avm_fritz5490.dtsi

```diff
@@ -0,0 +1,127 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "vr9_avm_fritzxx90.dtsi"
+
+/ {
+	compatible = "avm,fritz5490", "lantiq,xway", "lantiq,vr9";
+	model = "AVM FRITZ!Box 5490/5491";
+
+	remoteproc-wasp {
+		compatible = "avm,fritzbox5490-wasp";
+		mdio-device = <&gswip_mdio>;
+		link-interface = <&port5>;
+		reset-gpios = <&gpio 34 GPIO_ACTIVE_HIGH>;
+		power-gpios = <&gpio 5 GPIO_ACTIVE_HIGH>;
+	};
+};
+
+&aliases {
+	led-dsl = &led_info_green;
+	led-internet = &led_internet;
+	led-wifi = &led_wifi;
+};
+
+&leds {
+	led_fiber: fiber {
+		label = "green:fiber";
+		gpios = <&gpio 47 GPIO_ACTIVE_LOW>;
+	};
+
+	led_wifi: wifi {
+		label = "green:wlan";
+		gpios = <&gpio 36 GPIO_ACTIVE_LOW>;
+	};
+
+	led_internet: internet {
+		label = "green:internet";
+		gpios = <&gpio 35 GPIO_ACTIVE_LOW>;
+	};
+};
+
+&gswip_mdio {
+	phy5: ethernet-phy@5 {
+		reg = <0x05>;
+	};
+
+	phy6: ethernet-phy@6 {
+		reg = <0x06>;
+		reset-gpios = <&gpio 32 GPIO_ACTIVE_LOW>;
+	};
+
+	phy9: ethernet-phy@9 {
+		reg = <0x09>;
+	};
+
+	switch@0 {
+		compatible = "qca,qca8337";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		reset-gpios = <&gpio 44 GPIO_ACTIVE_LOW>;
+		reg = <0x0>;
+
+		dsa,member = <1 1>;
+
+		ports {
+			#address-cells = <1>;
+			#size-cells = <0>;
+
+			sw1port0: port@0 {
+				reg = <0>;
+				label = "dsa";
+				ethernet = <&sw0port1>;
+				phy-mode = "rgmii";
+
+				fixed-link {
+					speed = <1000>;
+					full-duplex;
+				};
+			};
+
+			port@2 {
+				reg = <2>;
+				label = "lan3";
+				phy-mode = "internal";
+			};
+
+			port@3 {
+				reg = <3>;
+				label = "lan4";
+				phy-mode = "internal";
+			};
+		};
+	};
+};
+
+&gswip_ports {
+	port@0 {
+		reg = <0>;
+		label = "wan";
+		phy-mode = "rgmii";
+		phy-handle = <&phy6>;
+	};
+
+	sw0port1: port@1 {
+		reg = <1>;
+		label = "lan-qca8k";
+		phy-mode = "rgmii";
+
+		fixed-link {
+			speed = <1000>;
+			full-duplex;
+		};
```

Comment:
Is the qca8k switch behind this port of the lantiq gswip?
Don't you  have to configure the qca8k switch here in device tree too?